inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -26,24 +28,28 @@ def __init__(self, sim_output_path: Path = None):
 
     def render_interactive(self,
                            representation: Literal["cartoon", "ball+stick",
-                                                   "line", "point",
+                                                   "line", "point", "surface",
                                                    "ribbon"] = "ball+stick",
+                           selection: str = "protein",
                            use_compressed_trajectory: bool = False,
                            add_backbone: bool = True):
         """
         Render the simulation outputs in an interactive visualization.
         Args: 
             representation: The protein representation to use for the 
             visualization.
+            selection: The selection to use for the representation. Check 
+            https://nglviewer.org/ngl/api/manual/usage/selection-language.html 
+            for details.
             add_backbone: Whether to add the protein backbone to the 
             visualization.
             use_compressed_trajectory: Whether to use the compressed 
             trajectory or the full precision one.
             """
         universe = self.construct_universe(use_compressed_trajectory)
-        view = nv.show_mdanalysis(universe)
-        view.add_representation(representation,
-                                selection="not water and not ion")
+        protein = universe.select_atoms("protein")
+        view = nv.show_mdanalysis(protein)
+        view.add_representation(representation, selection=selection)
         if add_backbone:
```

Comment:
explain why you have fixed "cartoon" here.

Reply:
Because I'm representing the backbone of the protein and I want it to stand out from the rest. In a photo is easier to see.

Reply:
<img width="322" alt="image" src="https://github.com/inductiva/inductiva/assets/114397668/269c435f-def1-4e70-8d09-04e0052ef4b9">
The blue part is the backbone and I want to highlight it!

Reply:
Great. But the reviewer may not know that. Please explain that because that's what you want, you are hardcoding "cartoon" there.